### DISCUSSION TITLE
macOS wine casks

### DISCUSSION THREAD

**Discussion by Gcenx** [Homebrew/discussions#2246] [Casks] [answered]
As most here will already be aware the current wine-* casks are installing the old depreciated macOS packages.   
Those can not be installed see https://github.com/Homebrew/homebrew-cask/pull/111086

@gverm and myself are now the official Winehq macOS package maintainers see https://www.winehq.org/pipermail/wine-devel/2021-July/191504.html   
We however are still working on some additional libraries but in the meantime would like to propose switching the current casks over to the packages I’m providing via GitHub.

The packages can be found here https://github.com/Gcenx/macOS_Wine_builds    
The my brew tap that provides these and some other casks can be found here https://github.com/Gcenx/homebrew-wine

<br>

Is there anything specific required to getting these casks swapped over until we provide the final packages directly via Winehq?

**Comment by SMillerDev** [comment, marked as answer]
The requirements for switching sources are that the original upstream has "blessed" the new repo, or that there is consensus on the package manager world that this is the new upstream.

**Reply by Gcenx** [reply]
I’m assuming the provided information isn’t enough to swap the casks over to the current WIP packages?

**Reply by carlocab** [reply]
The thread at https://www.winehq.org/pipermail/wine-devel/2021-July/191504.html looks good enough as "blessed" to me.

**Comment by owine** [comment]
Since this change, I am getting the following error with `wine-stable`.

`Error: Cask 'wine-stable' definition is invalid: invalid depends_on key: ':x11'`

I have confirmed that `:x11` does not appear in the cask file and have performed a `brew update-reset`. I can't even uninstall the `wine-stable` cask due to the same error.

**Reply by carlocab** [reply]
Try removing the cask manually and then doing 
```
rm -rf "$(brew --prefix)/Caskroom/wine-stable"
```

**Reply by cypok** [reply]
@carlocab, what do you mean by "removing the cask manually"? I have the same problem as @owine, can't upgrade or uninstall:
```
$ brew uninstall wine-stable 
brew list | Error: Cask 'wine-stable' definition is invalid: invalid depends_on key: ':x11'
```

**Comment by Gcenx** [comment]
Do I open a PR for each version bump?, I have build and updated wine-devel/staging-6.20 packages.

Also currently have wine-stable-6.0.2 buildings.

**Reply by SMillerDev** [reply]
Yeah, that's the idea

**Reply by Gcenx** [reply]
So one for wine-devel and another for wine-staging got it, I’ll open those later.
